Final Project/scraperTest.py

```diff
@@ -1,40 +1,72 @@
 """
 unit tests for scraper
 """
-
 #import response
 import unittest
-
+import requests
+import csv
+import os.path
+import re
+import time
+from bs4 import BeautifulSoup
+from selenium import webdriver
 import scraper as myScraper
 
+dataFilename='trainingCards.txt'
+url = myScraper.randomLinkGenerator()
+response = myScraper.requestBuilder(url)
+
 def checkDataParsedTest():
     assert myScraper.checkDataParsed('authors.txt') == True
 
 def requestBuilderTest():
     assert myScraper.requestBuilder('http://www.google.com').status_code == 200
 
 def driverBuilderTest():
-    #TODO
-    #Assert Driver is built
+#   TODO
+#   Assert Driver is built
```

Comment:
Can you remove the todo comments when they are finished ``

Reply:
Done